main.go

```diff
@@ -32,9 +32,10 @@ var (
 	flagNoSquash = flag.Bool("nosquash", false, "Don't squash layers and output every layer as ACI")
 	flagImage    = flag.String("image", "", "When converting a local file, it selects a particular image to convert. Format: IMAGE_NAME[:TAG]")
 	flagDebug    = flag.Bool("debug", false, "Enables debug messages")
+	flagInsecure = flag.Bool("insecure", false, "Allows unencrypted connections when fetching images")
```

Comment:
I'll change the wording, but the server could still redirect to an https connection.